assets/aml-benchmark/components/src/benchmark_result_aggregator/main.py

```diff
@@ -209,8 +209,9 @@ def _get_pipeline_params() -> Tuple[
 def _log_params_and_metrics(parameters: Dict[str, Any], metrics: Dict[str, Any]) -> None:
```

Comment:
Can we bump the version [over here](https://github.com/Azure/azureml-assets/blob/1cea65e5370e83214380d468fed6992db2d9f2f6/assets/aml-benchmark/components/benchmark-result-aggregator/spec.yaml#L7) to `0.0.2`?

Reply:
Updated. Thanks!